main.py

```diff
@@ -4,9 +4,6 @@
 from discord.ext import commands
 from dotenv import load_dotenv
 
-# Creates the db if it doesn't exist
```

Comment:
Because now the database is created in `database/schema.py` and `database` is imported from plenty of other places so it's extraneous to also import it here (especially because the import is unused).

Reply:
Gotcha